tests/ui/package.json

```diff
@@ -9,11 +8,20 @@
     "ts-node": "^10.9.1"
   },
   "devDependencies": {
-    "@playwright/test": "^1.41.2"
+    "@cucumber/cucumber": "^10.3.1",
+    "@playwright/test": "^1.41.2",
+    "allure-commandline": "^2.27.0",
+    "allure-cucumberjs": "^2.12.0",
+    "allure-js-commons": "^2.12.0",
+    "allure-playwright": "^2.12.0"
   },
   "scripts": {
+    "build": "tsc -p tsconfig-build.json",
```

Comment:
Is there anything wrong with using the report.js file and not converting it?

Reply:
Nothing really. I assumed you wanted it as a .ts file as all other files are .ts.